### PR TITLE
navbar btns

### PR DESCRIPTION
**Pull request by Cindyzhang977**
Make `BackButton` component so that the links to go back are more noticeable. 

image

Style sign in button.
image

Add ability to sign out on route view and icons to buttons.
image

### CODE REVIEW

File: frontend/src/RouteView.js [hunk 1]
```diff
@@ -110,13 +112,9 @@ function RouteView({ loggedIn }) {
...
       </Modal>
-      <Button
-        variant="link"
-        className={styles.editButton}
-        onClick={() => handleRouting(history)}
-      >
-        Edit Attractions
-      </Button>
+      <div className={styles.editButton} onClick={() => handleRouting(history)}>
+        <BackButton text={'Edit Attractions'} />
```
Comment (genedwards) [inline frontend/src/RouteView.js:116]: nit: `text="Edit Attractions"`

Comment (Cindyzhang977) [reply frontend/src/RouteView.js]: It looks like prettier set it back to single quotes

**Comment by Cindyzhang977** [comment]
updated back btn: 
image

File: frontend/src/RouteView.js [hunk 2]
```diff
@@ -103,13 +104,9 @@ function RouteView({ loggedIn }) {
...
-      <Button
-        variant="link"
-        className={styles.editButton}
-        onClick={() => handleRouting(history, 'explore', tripObject, attractions)}
-      >
-        Edit Attractions
-      </Button>
+      <div className={styles.editBtnContainer}>
+        <BackButton onClick={() => handleRouting(history)} text={'Edit Attractions'} />
+      </div>
```
Comment (genedwards) [inline frontend/src/RouteView.js:109]: can we try
```JS
<BackButton className={styles.editBtnContainer} ...

// BackButton
function BackButton({ text, onClick, className }) {
  return (
    <Button className={`${styles.btn} ${className}`} ...
```

File: frontend/src/RouteView.js [hunk 3]
```diff
@@ -103,13 +104,9 @@ function RouteView({ loggedIn }) {
...
       </Modal>
-      <Button
-        variant="link"
-        className={styles.editButton}
-        onClick={() => handleRouting(history, 'explore', tripObject, attractions)}
-      >
-        Edit Attractions
-      </Button>
+      <div className={styles.editBtnContainer}>
+        <BackButton onClick={() => handleRouting(history)} text={'Edit Attractions'} />
```
Comment (genedwards) [inline frontend/src/RouteView.js:108]: nit: `text="Edit Attractions"`

Comment (genedwards) [inline frontend/src/RouteView.js:108]: don't you need to pass the other arguments to `handleRouting(history, 'route', tripObject, attractions)`?

great example of where unit tests would have helped

Comment (Cindyzhang977) [reply frontend/src/RouteView.js:108]: oh yeah the PRs are out of order. I changed this in a later PR that already got merged into master.

**Pull request merged**